src/content/user-guide/arm-alpha.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: "DeepSparse ARM Alpha"
+metaTitle: "DeepSparse ARM Alpha"
+metaDescription: "Description of how to trial DeepSparse on ARM"
+index: 6000
+---
+
+# DeepSparse ARM Alpha
+
+DeepSparse has Alpha support for ARM platforms! This page explains how to trial it.
+
+[Sign up for early access to the general release of DeepSparse on ARM](https://neuralmagic.com/deepsparse-arm-waitlist)
+
+## Want to Particpate in Our Trial?
+
+Neural Magic is offering a $50 Amazon Gift card for a 30 minute conversation to discuss: 
+- Experience with the ARM Alpha
+- Requests for features for DeepSparse on ARM
+- Requests for ARM platform support (e.g. Raspberry Pi, Rockchip, Qualcomm)
+
+[Reach out to set up time to discuss](https://neuralmagic.com/contact/)
+
+## Understand Alpha Status
+
+DeepSparse on ARM is currently in Alpha state. 
+
+It has the following limitations:
+- **Long Compile Times** (compiling BERT on `M6g.16xlarge` takes XXX minutes)
+- **Slower Performance than DeepSparse x86** 
+- **No Performance Gains from Quantization**
+- **Limited Set of Models** (we have tested ResNet-50, YOLOv5, YOLACT, and BERT)
+- **AWS Graviton Only**
+
```

Comment:
If the items above are not titles or proper names, only the first word should be capitalized. For example: Slower performance than DeepSparse x86